requirements/index.html

```diff
@@ -0,0 +1,289 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
+	<head>
+		<title>Requirements for WCAG 3.0</title>
+		<meta charset="UTF-8" />
+		<script src="respec-config.js" class="remove"></script>
+		<script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"></script>
+        <script src="respec-config.js" class="remove"></script>
+	</head>
+	<body>
+		<section id="abstract">
+			<p>The Requirements for W3C Accessibility Guidelines (WCAG) 3.0 documentation is the next phase of development of the next major upgrade to accessibility guidelines. WCAG 3.0 will be the successor to the Web Content Accessibility Guidelines (WCAG) 2 series. The Silver Task Force of the Accessibility Guidelines Working Group and the W3C Silver Community group have partnered to incubate the needs, requirements, and structure for the new accessibility guidance. To date, the group  has: </p>
+			<ol>
+			  <li>Researched accessibility guidance needs.</li>
+			  <li>Developed problem statements and opportunities to improve accessibility guidance.</li>
+			  <li>Received input from industry leaders for directions to proceed.</li>
+			  <li>Drafted these high-level requirements for the next phase of the project, the prototyping and public input. </li>
+          </ol>
+    </section>
+		<section id="sotd">
+			<p>The Requirements for WCAG 3.0 is published as a joint effort of the <a href="https://www.w3.org/WAI/GL/task-forces/silver/wiki/Main_Page">Silver Task Force</a> of the <a href="https://www.w3.org/WAI/GL/">Accessibility Guidelines Working Group</a> and of the <a href="https://www.w3.org/community/silver/">W3C Silver Community Group</a>. It is a work in progress, and comments are welcome as <a href="https://github.com/w3c/silver/issues">Github Issues</a> or by <a href="mailto:public-silver@w3.org?subject=Comment%20Requirements%20for%20Silver">email</a>. </p>
+			<p>This is the second phase of the Requirements developed during Q1 2019 after months of prototyping work. We anticipate one additional phase in Q3-Q4 2019 after months of evaluating how well actual content can be developed for WCAG 3.0.  </p>
```

Comment:
This talks to 2019 and past tense, do we want to update here or are you initially transferring then updating to past tense?